### ISSUE TITLE
end is too aggressively dedented

### ISSUE THREAD

**Issue by chris-b1** [julia-vscode/julia-vscode#2068]
Possibly related to/duplicate of #1938, #2059

simple repro
```julia
function thing(a)
end_date = 1  # <---- should be indented
end
```

As soon as I type `end`, it is automatically de-dented back to column 1 - I think it should instead wait for a newline or whitespace to apply that?

**Comment by pfitzseb** [comment]
No, this is a separate issue. The ones you linked are only relevant to the formatter, which is entirely different from the formatting going on while typing.

**Comment by CameronBieganek** [comment]
Hmm, I see the problem, but I do kind of like how `end` is immediately de-dented after I type `d` ...

**Comment by pfitzseb** [comment]
I don't currently see a way to fix this without introducing other (worse, imho) edge cases.

**Comment by davidanthoff** [comment]
Closing: already marked `wontfix` — no fix was found that does not introduce worse indentation edge cases elsewhere.
